packages/desktop/src/services/FileDownloadServiceElectron.ts

```diff
@@ -15,6 +15,8 @@ import {
     HttpServiceBase,
 } from "../../../core/services";
 import { DownloadFailure } from "../../../core/errors";
+import axios from "axios";
```

Comment:
These should go in the third party imports section alphabetically (by `cockatiel` and `electron`

Reply:
resolved in [cfdf9b4](https://github.com/AllenInstitute/biofile-finder/pull/204/commits/cfdf9b40628c86077ac4ad73f158267bde3b5b49)